File: branches/Andean/Fantasy.Base/XSerialization/XException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.XSerialization
{
    [Serializable ]
    public class XException : Exception 
    {
        public XException(string message)
            : base(message)
        {

        }

        public XException()
        {

        }

        public XException(string message, Exception innerException)
            : base(message, innerException)

        {

        }
    }
}

File: AutoConverted/Fantasy.Base/XSerialization/XException.java
package Fantasy.XSerialization;

public class XException extends RuntimeException implements Serializable
{
	public XException(String message)
	{
		super(message);

	}

	public XException()
	{

	}

	public XException(String message, RuntimeException innerException)

	{
		super(message, innerException);

	}
}